animatch/EditProfile/EditProfile1.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  EditProfile1.swift
+//  animatch
+//
+//  Created by Ксения Панкратова on 18.11.2024.
+//
+
+import SwiftUI
+
+
+
+struct EditProfile1: View {
+    var body: some View {
+        VStack {
+            ZStack{
+                LinearGradient(gradient: Gradient(colors: [.editProfLightGrad, .editProfDarkGrad]), startPoint: .top, endPoint: .center)
+                    .ignoresSafeArea()
+             
+                VStack(spacing: 0){
+                    
+                    Head()
+                    
+                    Spacer()
+                    
+                    Forms()
+                    
+                    //у полей ввода нижние паддинги 30
+                        .padding(.bottom, 124)
+                    VStack(spacing: 0){
+                        
+                        ZStack{
+                            CustomButton(placeholder: "Сохранить и продолжить")
+                        }
```

Comment:
ZStack тут не нужен, у тебя же одна вьюха

```suggestion
                        CustomButton(placeholder: "Сохранить и продолжить")
```